ckanext/validation/plugin/__init__.py

```diff
@@ -145,9 +145,10 @@ def _process_schema_fields(self, data_dict):
         schema_upload = data_dict.pop(u'schema_upload', None)
         schema_url = data_dict.pop(u'schema_url', None)
         schema_json = data_dict.pop(u'schema_json', None)
-        if isinstance(schema_upload, ALLOWED_UPLOAD_TYPES) and schema_upload.content_length > 0:
```

Comment:
The reason to check content length is because a choice not to upload is sometimes represented as a zero-length file. Taking out the check might mean that a request is incorrectly treated as an upload when it isn't one.

Reply:
This was fixed in d0f780